README.md

```diff
@@ -42,6 +42,18 @@ Build instructions and more details can be found in the
 * Step 1: Make a space flight simulator
 * Step 2: Bloat it with features
 
+## Compiling for Windows on Visual Studio
+You will need: cmake, Visual Studio
+* Clone the repo with `https://github.com/TheOpenSpaceProgram/osp-magnum`
```

Comment:
You can re-write these three lines as

* Clone the repo with `git clone --recursive --shallow-submodules https://github.com/TheOpenSpaceProgram/osp-magnum`
* Navigate to the repo `cd osp-magnum`